update.sh

```diff
@@ -17,6 +17,8 @@ packages="$packages e2fsprogs"
 packages="$packages f2fs-tools"
 packages="$packages gpgv"
 packages="$packages raspbian-archive-keyring"
+packages="$packages wpasupplicant"
+packages="$packages firmware-realtek"
```

Comment:
Will do. However, I'll ignore the "various others" unless you have some specific list.
What to include here is part of the "what to do with drivers"-discussion.